templates/v2/user/_beneficiary_settings_form.html.twig

```diff
@@ -0,0 +1,28 @@
+{% form_theme form 'v2/form_theme/create_user_form_theme.html.twig' %}
+
+{{ form_start(form, {attr: {
+    'class': 'display-required-labels',
+    'data-controller': 'conditional-field',
+}}) }}
+    <div class="row bold">
+            {{ include('v2/user/_settings_form_block.html.twig', {
+                title: 'Informations',
+                leftColumnFields: [form.user.prenom, form.user.telephone],
```

Comment:
I am not sure you need these 2 columns, with a good flex wrap, you can get around this easily